nasa-gibs/worldview
# WV-2740-npm-scans

## Description

Execute NPM scans and address vulnerabilities:

## How To Test

1. `git checkout WV-2740-npm-scans`
2. `npm ci`
3. `npm audit`
4. Verify 0 vulnerabilities

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
